.pipelines/containerSourceData/scripts/BuildGoldenContainer.sh

```diff
@@ -322,16 +324,31 @@ function finalize {
     echo "$GOLDEN_IMAGE_NAME_FINAL" >> "$OUTPUT_DIR/PublishedContainers-$IMAGE.txt"
 }
 
+function oras_attach {
+    local image_name=$1
+    oras attach \
+        --artifact-type "application/vnd.microsoft.artifact.lifecycle" \
+        --annotation "vnd.microsoft.artifact.lifecycle.end-of-life.date=$END_OF_LIFE_1_YEAR" \
```

Comment:
vnd?

Reply:
Yes, it is `vnd`. See this wiki: https://msazure.visualstudio.com/ContainersSupplyChain/_wiki/wikis/ContainersSupplyChain.wiki/636125/Adding-Image-Lifecycle-Metadata-in-Pipelines.